### PR TITLE
Screen adjustment and update translation to Spanish 

### PR DESCRIPTION
**Pull request by samforo**
Screen adjustment, is need check.

### CODE REVIEW

File: brevent/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -121,13 +121,13 @@
     <string name="brevent_auto">Auto Brevent</string>
     <string name="brevent_auto_label">Actualizar la lista de Brevent cuando las aplicaciones se instalen o desinstalen.</string>
     <string name="brevent_method">Método Brevent</string>
-    <string name="brevent_method_standby_forcestop">Standby, después forzar detener</string>
+    <string name="brevent_method_standby_forcestop">Standby,después forzar detener</string>
```
Comment (liudongmiao) [inline brevent/src/main/res/values-es/strings.xml:124]: I think, there should be a space.

**Review by liudongmiao — CHANGES_REQUESTED** [review]

File: brevent/src/main/res/values-es/strings.xml (hunk 1 above)
Comment (samforo) [reply brevent/src/main/res/values-es/strings.xml:124]: Without the space or without the point  could enter a line.  That's why I suggest change. Thanks in advance.

**Comment by samforo** [comment]
img-20170828-wa0000

Please check capture.

File: brevent/src/main/res/values-es/strings.xml (hunk 1 above)
Comment (liudongmiao) [reply brevent/src/main/res/values-es/strings.xml:124]: Please make a space. There are other screen-width devices.

**Review by liudongmiao — CHANGES_REQUESTED** [review]

**Comment by samforo** [comment]
I have tested it on:

Samsung Galaxy S5
Samsung Galaxy S7 edge
Samsung Galaxy S6 edge plus 
Sony Xperia Z5
Sony Xperia XZ
Xiaomi Redmi Note 4x

The result is the same, does not enter the screen and remains in two lines.

 You can remove the comma "," or also this option. It can be left like this and is still valid and clear:
<string name="brevent_method_standby_forcestop">Standby, luego forzar detener</string>

Después = after
Luego = then

If it's ok with you. A greeting

**Comment by samforo** [comment]
Hi, any news?

**Comment by liudongmiao** [comment]
> This branch has conflicts that must be resolved

Please fix it.

**Comment by samforo** [comment]
Ok, thanks

**Pull request closed without merging**